### PR TITLE
Update for Elm 0.19

### PR DESCRIPTION
**Pull request by eeue56**

### CODE REVIEW

File: src/Expect.elm [hunk 1]
```diff
@@ -362,12 +362,12 @@ within tolerance a b =
 {-| Passes if (and only if) a call to `within` with the same arguments would have failed.
 -}
 notWithin : FloatingPointTolerance -> Float -> Float -> Expectation
-notWithin tolerance a b =
```
Comment (mgold) [inline src/Expect.elm]: Let's change this for `within` as well.

File: src/Expect.elm [hunk 2]
```diff
@@ -394,6 +394,7 @@ true : String -> Bool -> Expectation
 true message bool =
     if bool then
         pass
+
```
Comment (mgold) [inline src/Expect.elm]: I'm assuming these are added by elm-format.

File: src/Fuzz.elm [hunk 3]
```diff
@@ -45,8 +46,8 @@ import Fuzz.Internal as Internal
         , invalidReason
         )
 import Lazy
-import Lazy.List exposing ((+++), LazyList)
-import Random.Pcg as Random exposing (Generator)
+import Lazy.List exposing (LazyList, append)
+import Random exposing (Generator)
```
Comment (mgold) [inline src/Fuzz.elm]: Let's also update the docs around line 65 of this file to refer to the new library.

File: src/Test.elm [hunk 4]
```diff
@@ -389,7 +395,7 @@ fuzz2 fuzzA fuzzB desc =
         fuzzer =
             Fuzz.tuple ( fuzzA, fuzzB )
     in
-    uncurry >> fuzz fuzzer desc
+    (\f ( a, b ) -> f a b) >> fuzz fuzzer desc
```
Comment (mgold) [inline src/Test.elm]: `(\f (a,b) -> fuzz fuzzer desc (f a b))`?

File: src/Test.elm [hunk 5]
```diff
@@ -412,63 +418,10 @@ fuzz3 fuzzA fuzzB fuzzC desc =
     uncurry3 >> fuzz fuzzer desc
 
 
-{-| Run a [fuzz test](#fuzz) using four random inputs.
-
-This is a convenience function that lets you skip calling [`Fuzz.tuple4`](Fuzz#tuple4).
-
--}
-fuzz4 :
-    Fuzzer a
```
Comment (mgold) [inline src/Test.elm]: This is the first I've heard of this API change. I'm okay with it, but I'm not sure it should be bunched into the 0.19 upgrade.

We have some other major changes we want to make, and we should also have a version where these functions are deprecated.

Comment (avh4) [reply src/Test.elm]: Tuples of size 4 and larger are disallowed by elm-compiler in 0.19, so uncurry4/5 and tuple4/5 have to be removed.  (Though I suppose fuzz4/5 could technically stay, but would be slightly messy to reimplement without tuple/uncurry.)

Comment (mgold) [reply src/Test.elm]: On second thought, I'm okay with one big release. Let's just be sure to look at the *major release blocker* keyword on the old repo

Comment (mgold) [reply src/Test.elm]: Makes sense. fuzz4/5 are also not great since shrinking is hard.

File: src/Fuzz.elm (hunk 3 above)
Comment (mgold) [reply src/Fuzz.elm]: 3b070c1

File: src/Expect.elm (hunk 1 above)
Comment (mgold) [reply src/Expect.elm]: 3b070c1

**Comment by mgold** [comment]
I think we're good for an alpha release to get other packages unblocked.

**Comment by eeue56** [comment]
Not ready for merge/publishing

**Comment by avh4** [comment]
#2 adds on to this branch

**Comment by avh4** [comment]
From slack:

> eeue56 [12:23 PM]
> I think we need to merge elm-html-test at some point, but it probably doesnt need to be before 1.0.0
> 
> (we can merge it before the non-alpha)
> But actually now the majority of stuff to do is the CLI, which Richard is looking at

**Pull request merged**